src/Helper/FasitHelper.php

```diff
@@ -317,7 +317,7 @@ private function uploadFile(string $originalFilename, string $tempFilename): arr
     try {
       $options = [
         'headers' => [
-          'Content-Type' => 'multipart/form-data',
+          'Content-Type' => 'application/pdf',
```

Comment:
But you're sending `multipart` data (cf. line 324)!?

Reply:
Fixed.